src/test/java/com/artipie/rpm/RpmITCase.java

```diff
@@ -91,31 +115,70 @@ void addsRpmAndCreatesRepodata() throws Exception {
         );
         MatcherAssert.assertThat(
             "Repository xml indexes are created",
-            new FileStorage(this.tmp).list(new Key.From("my-rpm/repodata")).join().size(),
+            new FileStorage(this.tmp).list(new Key.From(RpmITCase.REPO, "repodata")).join().size(),
             new IsEqual<>(4)
         );
         con.disconnect();
     }
 
+    @Test
+    void listAndInstallsRpmPackage() throws Exception {
```

Comment:
@olenagerasimova is there reason to test install and list operations in single test? I'd suggest to create separate test for these operations